### ISSUE TITLE
doesn't work when first word in the line is tag name

### ISSUE THREAD

**Issue by rostowner** [johnotander/gulp-class-prefix#1]
### expected:

```
.PREFIX-btn-danger[disabled],
fieldset[disabled] .PREFIX-btn-danger,
.PREFIX-btn-danger.PREFIX-disabled:hover,
```
### actually:

```
.PREFIX-btn-danger[disabled],
fieldset[disabled] .btn-danger,
.PREFIX-btn-danger.PREFIX-disabled:hover,
```

**Comment by johno** [comment]
Thanks for opening this issue, @rostowner. I'll have some time this afternoon to work on a fix. I'll keep you posted.

**Comment by johno** [comment]
I've just published version `1.0.0` which should fix prefixing nested classes. Thanks for reporting!
